NSS-Bashful-Turtles/Yak-Social-Network
# Search functionality

added functionality to the search bar

to test
```
git fetch --all
git checkout rm-search
```

1. make sure api and npm dev server are running
1. load up web page and check to make sure the new drop down next to search bar is working
1. first try a search with no drop down option selected, you should get an error displayed but no react or build errors
1. now try searching for people, events, and posts. A basic event component has been created though may need to be edited once we build the calendar component
1. review code. most new code is in new search directory
